tests/test_deprecated.py

```diff
@@ -58,6 +62,18 @@ def test_tbd_remove_in_v0_9_0_module_imports():
     from pytorch_lightning.logging.wandb import WandbLogger  # noqa: F402
 
 
+def test_tbd_remove_in_v0_9_0_trainer():
+    mapping_old_new = {
+        'fast_dev_run': 'unit_test',
+    }
+    # skip 0 since it may be interested as False
+    kwargs = {k: (i + 1) for i, k in enumerate(mapping_old_new)}
```

Comment:
in this case, it is `bool` and the previous test tended to an arbitrary number...
consider changing/update to work with bools

Reply:
you mean to set it to True/False instead of int right ?

Reply:
you are right, as it is just one param, there is no need for this kind of heavy loop